examples/postgresql-replication/README.md

```diff
@@ -0,0 +1,36 @@
+The following directory contains examples of setting up a PostgreSQL master/standby cluster that is used for Sensu Go backend events.
+
+The example `sensu-backend.pp` will be applied to the Sensu Go backend.
+
+The example `postgersql.pp` will be applied to both the master and standby PostgreSQL servers.
+
+Adjustments will have to be made for IP addresses and password.
+
+Once the master and standby have applied their Puppet catalog you must bootstrap the standby:
+
+```
+systemctl stop postgresql-9.6.service
+rm -rf /var/lib/pgsql/9.6/data/*
+sudo -u postgres pg_basebackup -h 192.168.52.11 -D /var/lib/pgsql/9.6/data -P -U repl -R --xlog-method=stream
```

Comment:
Do you have to use IP's here or can you just use the FQDN?

Reply:
The documentation I found used IPs, but I would think FQDN works too. I used IPs because it was easier in testing with Vagrant.